File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int row = 0;
        int col = 0;
        for(int i = 0; i < sizes.length; i++) {
            int tmp = sizes[i][0] >= sizes[i][1] ? 0 : 1;
            if(row < sizes[i][tmp]) {
                row = sizes[i][tmp];
            }
            if(col < sizes[i][1 - tmp]) {
                col = sizes[i][1 - tmp];
            }
        }
        return row * col;
    }
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    var [min, max] = [0, 0];
    for(let i = 0; i < sizes.length; i++) {
        min = Math.max(min, Math.min(sizes[i][0], sizes[i][1]));
        max = Math.max(max, Math.max(sizes[i][0], sizes[i][1]));
    }
    return min * max;
}